daprdocs/content/en/operations/components/component-secrets.md

```diff
@@ -18,40 +18,9 @@ When running in Kubernetes, if the `auth.secretStore` is empty, the Kubernetes s
 
 Go to [this]({{< ref "howto-secrets.md" >}}) link to see all the secret stores supported by Dapr, along with information on how to configure and use them.
 
-## Non default namespaces
-
-If your Dapr enabled apps are using components that fetch secrets from non-default namespaces, apply the following resources to the namespace:
-
-```yaml
----
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  name: secret-reader
-  namespace: <NAMESPACE>
-rules:
-- apiGroups: [""]
-  resources: ["secrets"]
-  verbs: ["get"]
----
-
-kind: RoleBinding
-apiVersion: rbac.authorization.k8s.io/v1
-metadata:
-  name: dapr-secret-reader
-  namespace: <NAMESPACE>
-subjects:
-- kind: ServiceAccount
-  name: default
-roleRef:
-  kind: Role
-  name: secret-reader
-  apiGroup: rbac.authorization.k8s.io
-```
-
 ## Examples
 
-Using plain text:
+Using plain text secrets (not recommended for production):
```

Comment:
This should be a ### header, and move `(not recommended for production)` into the next line as a warning:

```html
{{% alert title="Note" color="warning" %}}
Plaintext secrets are not recommended for production environments. Use a secret store instead to securly store and access keys.
{{% /alert %}}
```